### DISCUSSION TITLE
VirtualC64 Homebrew

### DISCUSSION THREAD

**Discussion by Haroldo64** [dirkwhoffmann/virtualc64#688] [Q&A]
Hi, I've seen that there is VirtualC64 here:

https://formulae.brew.sh/cask/virtualc64#default

Can someone tell me if this homebrew can be compiled for arm64 (Apple Silicon) or if this formula will generate only an Intel x86-64 code ?

**Comment by dirkwhoffmann** [comment]
VirtualC64 is arm64 compatible, but I don't know if the brew formula installs a universal binary.
